src/app/components/media/CreateRelatedMedia.js

```diff
@@ -39,6 +39,20 @@ class CreateRelatedMedia extends Component {
     this.setState({ dialogOpen: false });
   };
 
+  currentProject() {
+    let project = null;
+    let currentProjectId = window.location.pathname.match(/project\/([0-9]+)/);
+    if (currentProjectId) {
+      currentProjectId = parseInt(currentProjectId[1], 10);
+      project = this.props.media.projects.edges.filter(p =>
```

Comment:
Please use `find` or `findIndex` instead of `filter`

Reply:
Done